fpm_template.go

```diff
@@ -136,7 +139,8 @@ arch: {{ .Arch }}
 platform: linux
 release: 1
 version: {{ .Version }}
-maintainer: {{ .Maintainer }}
+maintainer: "HashiCorp"
```

Comment:
I think instead of hardcoding these here - we would want to add something to the above section for `inputVendor` and `inputMaintainer` to set these to `HashiCorp` unless a value has been passed in:
```
        inputVendor := os.Getenv("INPUT_VENDOR")
	if inputVendor == "" {
		inputVendor = "HashiCorp"
	}
```

Reply:
My personal opinion on this is we should set it as close to user input as possible so it is surfaced easier. So for example, rather than having the empty check in go code here, set the default in the input file https://github.com/hashicorp/actions-packaging-linux/pull/14/files#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R33. This action should primarily be for us at HashiCorp to use internally but it seems more intuitive that users would check the input file because that's their interface with the action, rather than dig into the template file to check what the implementation details are and find out we are overwriting empty strings.

Reply:
So the default is defined in GHA instead of the tool itself.

I like that. It also addresses the issue of opting out of the tag if one were to re-use the tool elsewhere.